examples/tutorials/core_api_mnist/adaptive.yaml

```diff
@@ -0,0 +1,33 @@
+name: coreapi_mnist_tutorial_adaptive_search
```

Comment:
# Use an adaptive ASHA search to optimize a hyperparameter

Reply:
Instead of a comment, that could be set as the `description:` field, that might be nice.

Then it would show up in the webui too